public/app/features/variables/pickers/PickerRenderer.tsx

```diff
@@ -1,33 +1,56 @@
-import React, { FunctionComponent, useMemo } from 'react';
+import React, { FunctionComponent, PropsWithChildren, ReactElement, useMemo } from 'react';
 import { VariableHide, VariableModel } from '../types';
 import { selectors } from '@grafana/e2e-selectors';
 import { variableAdapters } from '../adapters';
+import { Tooltip } from '@grafana/ui';
 
 interface Props {
   variable: VariableModel;
 }
 
 export const PickerRenderer: FunctionComponent<Props> = props => {
   const PickerToRender = useMemo(() => variableAdapters.get(props.variable.type).picker, [props.variable]);
-  const labelOrName = useMemo(() => props.variable.label || props.variable.name, [props.variable]);
 
   if (!props.variable) {
     return <div>Couldn't load variable</div>;
   }
 
   return (
     <div className="gf-form">
```

Comment:
nit: change to vertical group?

Reply:
👍

Reply:
Aha sorry I only went through the forms, not the pickers. So I'll not look into this now.